dotcom-rendering/src/components/FrontSection.tsx

```diff
@@ -179,27 +179,29 @@ const containerStyles = css`
 	}
 `;
 
-const sectionHeadline = css`
-	grid-row: headline;
-	grid-column: title;
+const sectionHeadline = (borderColour: string) => {
```

Comment:
You could write this like this if you wanted to be concise 😉 

```suggestion
const sectionHeadline = (borderColour: string) => css`
    (the css)
`;
```